jormungandr-lib/src/interfaces/fragments_processing_summary.rs

```diff
@@ -0,0 +1,55 @@
+use chain_impl_mockchain::fragment::FragmentId;
+
+use serde::{Deserialize, Serialize};
+use serde_with::{serde_as, DisplayFromStr};
+
+/// This error is reserved for fragments that were rejected by the mempool at the time of sending
+/// them to mempool. If a fragment ended up being included to mempool, it will be listed in
+/// fragment logs and all further errors would be listed in fragment logs as well.
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(tag = "type")]
```

Comment:
`tag` and `flatten`, should work with `Serialize`. We should have deserialization tests to verify it works on parsing.

The tag should be `reason` to match the API description, though?